From 5493ca7050e8285be75e439bdc69213eb3338f69 Mon Sep 17 00:00:00 2001
From: Marcin Owsiany <marcin@owsiany.pl>
Date: Wed, 28 Aug 2013 11:48:35 -0700
Subject: [PATCH 7/7] Bump cache TTL from 3h to 2 weeks.

---
 driver/xscreensaver-getimage-file | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/driver/xscreensaver-getimage-file b/driver/xscreensaver-getimage-file
index d3090f3..d46a958 100755
--- a/driver/xscreensaver-getimage-file
+++ b/driver/xscreensaver-getimage-file
@@ -74,7 +74,8 @@ my $cache_p = 1;
 
 # Regenerate the cache if it is older than this many seconds.
 #
-my $cache_max_age = 60 * 60 * 3;   # 3 hours
+my $cache_max_age = 60 * 60 * 24 * 7 * 2;   # 2 weeks
+
 
 # Re-poll RSS/Atom feeds when local copy is older than this many seconds.
 #
-- 
1.8.4

