#
# Sent upstream (jwz) 2009-09-27
#
Index: xscreensaver/hacks/glx/flurry.man
===================================================================
--- xscreensaver.orig/hacks/glx/flurry.man	2008-07-31 21:44:30.000000000 +0200
+++ xscreensaver/hacks/glx/flurry.man	2009-09-27 22:20:57.000000000 +0200
@@ -1,6 +1,6 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-flurry
+flurry \- a colourful star(fish)like flurry of particles
 .SH SYNOPSIS
 .B flurry
 [\-display \fIhost:display.screen\fP]
