Index: xscreensaver/driver/Makefile.in
===================================================================
--- xscreensaver.orig/driver/Makefile.in	2009-06-21 20:50:11.000000000 +0200
+++ xscreensaver/driver/Makefile.in	2009-06-21 20:50:28.000000000 +0200
@@ -258,7 +258,7 @@
 tests: $(TEST_EXES)
 
 install:   install-program   install-ad    install-scripts \
-	   install-gnome     install-man   install-xml install-pam
+	   install-gnome     install-man   install-xml
 uninstall: uninstall-program uninstall-ad  \
 	   uninstall-gnome   uninstall-man uninstall-xml  
 
